Merit_Visualization.js

```diff
@@ -0,0 +1,304 @@
+                      ':(b("hnd") > 30.0) ? 12' +
+                      ':(b("hnd") > 20.0) ? 11' +
+                      ':(b("hnd") > 15.0) ? 10' +
+                      ':(b("hnd") > 10.0) ? 9' +
+                      ':(b("hnd") > 7.0) ? 8' +
+                      ':(b("hnd") > 5.0) ? 7' +
+                      ':(b("hnd") > 2.0) ? 6' +
+                      ':(b("hnd") > 1.0) ? 5' +
+                      ':(b("hnd") > 0.7) ? 4' +
+                      ':(b("hnd") > 0.5) ? 3' +
+                      ':(b("hnd") > 0.1) ? 2' +
+                      ':1');
+
+//////////////////////////////////////////////////////////////
+// Set up the Interactive System on the Map
+//////////////////////////////////////////////////////////////
+
+// Set up the overall structure of the app, with a control panel to the left
+// of a full-screen map.
+ui.root.clear();
+var panel = ui.Panel({style: {width: '250px'}});
+var map = ui.Map();
+ui.root.add(panel).add(map);
+map.style().set('cursor', 'crosshair');
+
+// Create title
+panel.add(ui.Label(
+  'MERIT Hydro Visualization and Interactive Map',
+  {fontWeight: 'bold', fontSize:'20px'}
+));
+
+// Create a layer selector that dictates which layer is visible on the map.
+var select = ui.Select({
+  items: [RIVERCHANNEL, RIVERWIDTH, TERRAIN],
+  value: RIVERCHANNEL,
+  onChange: redraw,
+});
+panel.add(ui.Label('Visualization Map:')).add(select);
+
+// Check-boxes to control which layers are shown in the inspector.
+panel.add(ui.Label('Band Information:'));
+var wthCheck = ui.Checkbox(RIVERWIDTH).setValue(true);
+panel.add(wthCheck);
+var elevationCheck = ui.Checkbox(ELEVATION).setValue(true);
+panel.add(elevationCheck);
+var hndCheck = ui.Checkbox(HAND).setValue(true);
+panel.add(hndCheck);
+var upgCheck = ui.Checkbox(UPG).setValue(true);
+panel.add(upgCheck);
+var upaCheck = ui.Checkbox(UPA).setValue(true);
+panel.add(upaCheck);
+var dirCheck = ui.Checkbox(FLOWDIRECTION).setValue(true);
+panel.add(dirCheck);
+
+// Create the inspector panel, initially hiding it.
+var inspector = ui.Panel({style:{shown: false}});
+inspector.style().set({width: '250px', position: 'bottom-right'});
+map.add(inspector);
+
+// Register an onClick handler that populates and shows the inspector panel.
+map.onClick(function(coords) {
+  // Gather the image bands into a single Image that we can asynchronously sample.
+  
+  var point = ee.Geometry.Point(coords.lon, coords.lat);
+
+var lon = ui.Label();
+var lat = ui.Label();
+
+  // Add the dot as the second layer, so it shows up on top of the composite.
+  var dot = ui.Map.Layer(point, {color: 'FF0000'}, 'clicked location');
+  map.layers().set(1, dot);
+  
+  var sample = merit.unmask(0).sample(point, 30).first().toDictionary();
+  sample.evaluate(function(values) {
+    inspector.clear();
+    // Display a label that corresponds to a checked checkbox.
+    if (wthCheck.getValue()) {
+      inspector.add(ui.Label('River Width: ' + values.wth + ' m'));
```

Comment:
round the values to integer (maybe except for upstream drainage area which should be rounded to 2-3 decimal points)